concise-mid-tag.cddl

```diff
@@ -55,14 +55,11 @@ claims-map = non-empty<{
   ? comid.reference-claims => one-or-more<reference-claim-map>
   ? comid.endorsed-claims => one-or-more<endorsed-claim-map>
   ? comid.identity-claims => one-or-more<identity-claim-map>
-  ? comid.reference-instance-claims => one-or-more<reference-instance-claim-map>
-  ? comid.endorsed-instance-claims => one-or-more<endorsed-instance-claim-map>
   * $$claims-map-extension
 }>
 
 identity-claim-map = {
   ? comid.element-name => element-name-map
```

Comment:
element-name is the 'key' while device-id is one of the possible 'value' components in a key-value pair. Not sure why it makes sense to move it from the 'value' side to the 'key' side?